app/web/components/Navigation/Navigation.tsx

```diff
@@ -565,6 +565,7 @@ export default function Navigation() {
           </div>
         </Hidden>
       </Toolbar>
+      <GlobalMessage />
```

Comment:
I think it would be more efficient to fetch the data here, then use it to conditionally render `<GlobalMessage/>` and pass what you need in the props.

Otherwise you're gonna do an api call every time the `Navigation` loads.

Reply:
I've added fetch rate limiting into the component.